### ISSUE TITLE
Is automapping to a concrete class gone?

### ISSUE THREAD

**Issue by kdurnoga** [davidmoten/rxjava2-jdbc#15]
I've just migrated from rxjava-jdbc to rxjava2-jdbc and noticed that [automapping to a concrete class feature](https://github.com/davidmoten/rxjava-jdbc#automap-using-a-concrete-class) is missing. That was a really useful tiny little thing. Can we hope it returns at some point?

**Comment by davidmoten** [comment]
Yeah I did leave it out intentionally mainly because of type safety. If your constructor changes then you get no compile time indication and the complication of multiple constructors made the approach a bit unattractive to me. Probably your easiest workaround is to use an intermediate Tuple:

```java
db.select("select name, score from person")
  .getAs(String.class, Integer.class)
  .map(x -> new Person(x._1, x._2));
```

**Comment by kdurnoga** [comment]
I see, a pity. Working with tuples becomes cumbersome when the number of columns grows and you have to explicitly specify the type of each of them. Is it possible to reintroduce this functionality, possibly in some utility class?

**Comment by davidmoten** [comment]
So I'll mark it as an enhancement and pop it on my backlog. The code can be copied from `rxjava-jdbc` but I'd like to revisit some of the validation stuff (like the existence of multiple constructors with the same number of parameters for instance).

**Comment by kdurnoga** [comment]
Much appreciated!

**Comment by cosmin-ionita** [comment]
@davidmoten I tried to fix this issue here: https://github.com/davidmoten/rxjava2-jdbc/pull/53. Please let me know your thoughts!
